base/doc/ltnews40.tex

```diff
@@ -159,6 +159,28 @@ \section{Introduction}
 
 \emph{to write \ldots\ 30th aniversary of \LaTeXe{} this year btw}
 
+\section{Engine support: important update}
+
+As detailed below, work is progressing on the Tagged PDF project. There are
+many drivers for this work, including legal changes in many places which will
+increasingly require well-tagged PDFs including full support for mathematics. As
+part of the work on this, we are looking at the technical abilities of the
+\TeX{} engines.
+
+With \XeTeX{}, it is impossible to reliably produce tagged PDFs due to
+engine limitations. The increasing importance of tagged PDFs means that this
+requires a move away from \XeTeX{}. We will continue to address issues with
+\XeTeX{} support in team-maintained \LaTeX{} code on a best-effort basis.
+No \emph{new} functionality will be added for \XeTeX{} by the \LaTeX{} team. It
+is likely that over time functionality may become more restricted, and users
+are urged to migrate \XeTeX{} documents to \LuaTeX{}.
+
+For \pdfTeX{}, tagging is available and we are able to support mathematics by
+including \TeX{} mathematical sources or by using externally-generated MathML.
+Only \LuaTeX{} is capable of \emph{automatic} generation of MathML as part of a
+\LaTeX{} run. Thus \pdfTeX{} continues to be supported for existing material,
+but for new documents, moving to \LuaTeX{} is recommended.
+
```

Comment:
One could perhaps add a statement like

We cannot make statements about the support for other engines such as ... , as we don't use these programs nor have indepth knowledge of their functionalities. To the best of our knowledge, core LaTeX works well with these engines, but if and to what extent tagging can be supported will remain to be seen. If relevant information becomes available to us we will provide an update in future editions of the \LaTeX{} newsletter.